bgpd/bgp_route.c

```diff
@@ -13937,6 +13939,12 @@ DEFUN (show_bgp_afi_vpn_rd_route,
 		return CMD_WARNING;
 	}
 
+	if (!strcmp(argv[5]->arg, "all"))
```

Comment:
Is this valid if we specify `vpn all rd all`?

Reply:
@ton31337 logically or functionally? ;-)

Both `vpn all` and `rd all` are effectively equivalent keywords.  I did make sure that the two work together though:
```
ub20# show bgp ipv4 vpn all rd all
BGP table version is 3, local router ID is 100.64.0.222, vrf id 0
Default local pref 100, local AS 1
Status codes:  s suppressed, d damped, h history, * valid, > best, = multipath,
               i internal, r RIB-failure, S Stale, R Removed
Nexthop codes: @NNN nexthop's vrf id, < announce-nh-self
Origin codes:  i - IGP, e - EGP, ? - incomplete

   Network          Next Hop            Metric LocPrf Weight Path
Route Distinguisher: 1:1
*>i100.64.0.1/32    192.168.122.51           0    100      0 i
* i                 192.168.123.51           0    100      0 i
*>i100.64.0.111/32  192.168.122.51           0    100      0 i
*=i                 192.168.123.51           0    100      0 i
Route Distinguisher: 2:2
*> 100.64.0.2/32    0.0.0.0                  0         32768 i
*> 100.64.0.222/32  0.0.0.0                  0         32768 i
Route Distinguisher: 11:11
*>i100.64.0.11/32   192.168.122.51           0    100      0 i
*=i                 192.168.123.51           0    100      0 i
Route Distinguisher: 22:22
*> 100.64.0.22/32   0.0.0.0                  0         32768 i

Displayed  6 routes and 9 total paths
```

Reply:
:+1: